db/migrate/20160905122239_create_team_user.rb

```diff
@@ -1,8 +1,8 @@
 class CreateTeamUser < ActiveRecord::Migration[5.0]
   def change
     create_table :team_users do |t|
-      t.belongs_to :team, index: true
-      t.belongs_to :user, index: true
+      t.belongs_to :team, index: true, unique: true
+      t.belongs_to :user, index: true, unique: true
```

Comment:
this is not a composite index :p it will not work, you need to explicitly create an index `add_index :users, [:email, :account_id], unique: true` something like that